VERSION

```diff
@@ -1 +1 @@
-v0.5.2
+v0.5.3
```

Comment:
Since this is a feature change, please change the version as `v0.6.0`. We always use the PATCH part in the version (major.minor.patch) for bug fix.

Reply:
Done. Has changed the version as v0.6.0